ui/app/components/secret-edit.js

```diff
@@ -56,81 +62,75 @@ export default Component.extend(FocusOnInsertMixin, {
     }
     this.checkRows();
     if (this.get('wizard.featureState') === 'details' && this.get('mode') === 'create') {
-      let engine = this.get('key').backend.includes('kv') ? 'kv' : this.get('key').backend;
+      let engine = this.get('model').backend.includes('kv') ? 'kv' : this.get('model').backend;
       this.get('wizard').transitionFeatureMachine('details', 'CONTINUE', engine);
     }
 
-    if (this.get('mode') === 'edit') {
+    if (this.mode === 'edit') {
```

Comment:
looks like there are a few more `.get` statements you can update to the new style in here too

Reply:
thanks, fixed in the latest!